data/2022_23/seminare/P-MAT/P-MAT_sustredenie_leto_starsi.yml

```
@@ -0,0 +1,34 @@
+# Nazov udalosti
+name: letná sústredenie Pikomat starší
```

joj sorry ešte to máš zle vysklonovane

```suggestion
name: Letné sústredenie Pikomat starší
```